daml-script/runner/src/main/scala/com/digitalasset/daml/lf/engine/script/Runner.scala

```diff
@@ -479,9 +478,9 @@ class Runner(compiledPackages: CompiledPackages, script: Script.Action, timeMode
                         .toCommands(extendedCompiledPackages, payload.freeAp))
                       client <- Converter.toFuture(clients
                         .getPartiesParticipant(actAs))
-                      commitLocation <- payload.loc cata (sLoc =>
-                        Converter.toFuture(Converter.toOptionLocation(knownPackages, sLoc)),
-                      Future(None))
+                      commitLocation <- payload.loc.cata(
+                        sLoc => Converter.toFuture(Converter.toOptionLocation(knownPackages, sLoc)),
+                        Future(None))
```

Comment:
If you plan to make trailing commas the rule of the land, you may want to start now. This will be my final bike-shedding (for this week). 😛

Reply:
I mostly don’t want to insert any trailing commas and instead let scalafmt do it for me which I can’t atm :slightly_smiling_face: